tests/ea/resume.cpp

```diff
@@ -120,17 +120,20 @@ BOOST_AUTO_TEST_CASE(test_nsga2_resume) {
   typedef ea::Nsga2<phen_t, eval_t, stat_t, modifier_t, Params> ea_t;
   ea_t ea;
   ea.run();// 101 generations
+  std::cout<<"nsga2 done, running some tests..."<<std::endl;
```

Comment:
I think that Travis allows us to write to files almost everything..

But, it's true that we do not really care about verbose output/tests.. So, I am OK with it..